charts/seed-bootstrap/charts/elastic-kibana-curator/templates/kibana/kibana-deployment.yaml

```
@@ -18,23 +18,28 @@ spec:
       annotations:
         checksum/kibana-objects-registration-config: {{ include (print $.Template.BasePath "/kibana/kibana-objects-registration-config.yaml") . | sha256sum }}
         checksum/kibana-saved-objects-config: {{ include (print $.Template.BasePath "/kibana/kibana-saved-objects-config.yaml") . | sha256sum }}
+        checksum/kibana-config: {{ include (print $.Template.BasePath "/kibana/kibana-configmap.yaml") . | sha256sum }}
       labels:
         garden.sapcloud.io/role: logging
         app: kibana-logging
         role: logging
     spec:
       containers:
       - name: kibana-logging
+{{- if .Values.searchguard.enabled }}
+        image: {{ index .Values.global.images "kibana-searchguard-oss" }}
+{{- else -}}
```

```suggestion
{{- else }}
```

`-}}` trims the right whitespace (including `\n`) and the final result is `- name: kibana-loggingimage: [...]`.